newroutes/formsg.ts

```diff
@@ -0,0 +1,65 @@
+import { DecryptedContent, FormField } from "@opengovsg/formsg-sdk/dist/types"
+import autoBind from "auto-bind"
+import express, { Request, Response } from "express"
+
+import logger from "@logger/logger"
+
+import { BadRequestError } from "@errors/BadRequestError"
+
+import { attachFormSGHandler } from "@root/newmiddleware"
+import InfraService from "@services/infra/InfraService"
+
+const { SITE_CREATE_FORM_KEY } = process.env
+
+function getProp(responses: FormField[], name: string): string | undefined {
+  const response = responses.find(({ question }) => question === name)
+
+  return response?.answer
+}
```

Comment:
I think this could be moved to a util file to make this router class cleaner

Reply:
Yes, that's a bit cleaner. I'll do that.